### PR TITLE
[DOC-10197/release/3.1]: Bootstrap config hostnames must be of the Data Service Nodes specifically. Added warning note about using a node running the data service.

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-10197

### CODE REVIEW

File: modules/ROOT/pages/_partials/connection-node-warning.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+IMPORTANT: The connection must point to a node running the data service.
```
Comment (ElliotFrancisHunter) [inline modules/ROOT/pages/_partials/connection-node-warning.adoc]: Is this an admonition that will see frequent use in the docs? I'm not sure the abstraction is required for two instances

Comment (RayOffiah) [reply modules/ROOT/pages/_partials/connection-node-warning.adoc]: Probably not, actually; I think I'm only using it in two places. I tend to use includes when the text is going to appear in two or more places.

It's a habit from my programming days, I think. I can certainly bin it though.

Comment (RayOffiah) [reply modules/ROOT/pages/_partials/connection-node-warning.adoc]: No, hang on; just remembered why I labelled the branch with a version number. I'm going to use the same warning in the other versions.

I think I'll keep it as is.

Comment (RayOffiah) [reply modules/ROOT/pages/_partials/connection-node-warning.adoc]: Nope. I've changed my mind. It's overkill for one  single line of text, even if's going in multiple versions.

**Review by ElliotFrancisHunter — APPROVED** [review]
Looks good to me! 👍🏼

**Pull request merged**